src/contrib/mlir/compiler.cpp

```diff
@@ -322,15 +336,16 @@ void MLIRCompiler::lower_ng_dialect()
     pm.addPass(mlir::createDialectLoweringPass(this));
     pm.addPass(mlir::createCanonicalizerPass());
 
+    // Apply any generic pass manager command line options.
+    mlir::applyPassManagerCLOptions(pm);
+
     pm.run(m_module.get());
 
     if (failed(m_module->verify()))
     {
         NGRAPH_CHECK(false, "Incorrect module after dialect lowering");
     }
 
-    dump_mlir_module("Affine Dialect Dump (Pre-Optimizations):");
```

Comment:
Why removing this one ?

Reply:
Because it becomes redundant. It prints exactly the same as `-print-ir-after-all` for canonicalizer pass (line 323)

Reply:
Having two flags at the MLIR level can be confusing especially that we will likely mix ngraph and mlir passes (e.g. ngraph pass , mlir CSE, ngraph pass ...etc.). I would prefer us to print with `-print-ir-after-all` is ON instead.